src/main/java/com/netgrif/application/engine/pdf/generator/service/PdfGenerator.java

```diff
@@ -139,8 +139,8 @@ protected File transformRequestToPdf(List<PdfField> pdfFields, PdfResource pdfRe
     }
 
     protected void transformRequestToPdf(List<PdfField> pdfFields, PdfResource pdfResource, OutputStream stream) throws IOException {
-        File template = pdfResource.getTemplateResource().getFile();
-        if (template.exists()) {
+        if (pdfResource.getTemplateResource().exists()) {
+            InputStream template = pdfResource.getTemplateResource().getInputStream();
```

Comment:
Did you test it in different configurations? Because I remember that this part has been changed several times in the past.

Reply:
Yes, I tested this running from IDEA, running from JAR, with existing and non-existing files too. The PDBox library developers suggest to use inputstream instead of File, because there may have been issues resolving file path on different systems.